app/presenters/avatar_presenter.rb

```diff
@@ -6,7 +6,8 @@ def base_hash(with_default=false)
     avatar = {
       small:  small(with_default),
       medium: medium(with_default),
-      large:  large(with_default)
+      large:  large(with_default),
+      raw:    raw(with_default)
```

Comment:
Rails/OutputSafety: Tagging a string as html safe may be a security risk.

Reply:
That's a different `raw`, you funny bot!